tracer/test/Datadog.Trace.TestHelpers/TestHelper.cs

```diff
@@ -75,7 +75,7 @@ public Process StartDotnetTestSample(int traceAgentPort, string arguments, strin
                 traceAgentPort: traceAgentPort,
                 statsdPort: statsdPort,
                 aspNetCorePort: aspNetCorePort,
-                processToProfile: exec);
+                processToProfile: exec + ";testhost.exe");
```

Comment:
Interesting, why didn't we need this before? 🤔

Reply:
I guess VS2022??? 🤷🏻 But I was having trouble running the test + profiler inside the VS.